less/v2/apps.less

```diff
@@ -1113,16 +1114,27 @@ dialog.dnd5e2.application {
       border-radius: 0;
     }
     .name { grid-column: name; }
-    .proficiency { grid-column: proficiency; }
-    .mastery { grid-column: mastery; }
-    .proficiency, .mastery { text-align: center; }
+    .proficiency {
+      grid-column: proficiency;
+      text-align: center;
+    }
     li {
       align-items: center;
       &:not(:last-of-type) { border-bottom: var(--dnd5e-border-dotted); }
```

Comment:
Some style tweaks for the sub-category display. Works pretty well on smaller lists like Primordial here, but might not look great on longer lists, not sure:

![image](https://github.com/user-attachments/assets/79b1dfef-2c10-4072-a91c-ed8eccbabb72)

```suggestion
      &:not(:last-of-type) { border-bottom: var(--dnd5e-border-dotted); }
      &.category { gap: 4px; } 
      li {
        position: relative;
        margin: 0 0 4px 0;
        &::before {
          content: "┗";
          position: absolute;
          left: -1.8em;
          font-size: var(--font-size-10);
          font-weight: bold;
          top: 3px;
          color: var(--color-text-dark-5);
        }
      }
```